cabal.project.ci

```diff
@@ -1,34 +0,0 @@
-packages:
```

Comment:
What's the rationale behind removing this? I think one of the key goals is to enable `-Werror` for CI builds without requiring it for local builds.

Reply:
I removed it because having multiple files that are supposed to be identical except for one difference is always going to end up being error-prone. I see that this is the approach used by cabal's CI system, but I'm not sure I like it.

I did add back in the Werror by adding it into the CI after the cabal configure step. Not sure how much better that is; on one hand, it'll always be correct. On another, it's now somewhat buried inside the `.github` folder... I'm fine with either approach, ultimately, but I suspect this one might be better as I expect the cabal.project file to be edited more often than a `cabal.project.ci` file would be.

Reply:
Ah, okay. If this is being recreated by the Actions script, and always based on the original, that does seem better.